reframe/ds_wrapper.go

```diff
@@ -0,0 +1,128 @@
+package reframe
+
+import (
+	"bytes"
+	"context"
+	"encoding/binary"
+	"encoding/gob"
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-datastore"
+	dsq "github.com/ipfs/go-datastore/query"
+)
+
+const (
+	chunkByContextIdIndexPrefix = "reframe/ccid/"
+	timestampByCidIndexPrefix   = "reframe/tc/"
+)
```

Comment:
Instead of having the `reframe/` prefix explicitly, you can also use the datastore's namespace package:
https://pkg.go.dev/github.com/ipfs/go-datastore@v0.6.0/namespace
 ```go
 const reframeDSName = "reframe"

 dsWrapper: &dsWrapper{
	ds: namespace.Wrap(ds, reframeDSName),
},
```

Reply:
good point, added